File: Assignment_004/program5.c
#include<stdio.h>

int FactDiff(int iNo)
{
    int iCnt = 0;
    int iSum = 0;
    int iNum = 0;

    for(iCnt = 1; iCnt < iNo; iCnt++)
    {
        if((iNo % iCnt) == 0)
        {
            iSum = iSum + iCnt;
        }
        else if((iNo % iCnt) != 0)
        {
            iNum = iNum + iCnt;
        }
    }

    return iSum - iNum;

}
int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter the number\n");
    scanf("%d",&iValue);

    iRet = FactDiff(iValue);

    printf("%d\n",iRet);



    return 0;
}

File: Assignment_032/Program5.java
import java.util.Scanner;

class Bitwise
{
    public int ToggleBit(int iNo)
    {
        int iMask = 9;
        int iResult = 0;

        iResult = iNo ^ iMask;

       return iResult;
    }
}
class Program5
{
    public static void main(String A[])
    {
        int iValue = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the Number : ");
        iValue = sobj.nextInt();


        Bitwise bobj = new Bitwise();

        int iRet = 0;

        iRet = bobj.ToggleBit(iValue);

        System.out.println("Updated Number is : "+iRet);

    }
}